addons/purchase_requisition_stock/models/stock.py

```diff
@@ -45,7 +45,7 @@ class Orderpoint(models.Model):
     def _quantity_in_progress(self):
         res = super(Orderpoint, self)._quantity_in_progress()
         for op in self:
-            for pr in self.env['purchase.requisition'].search([('state', '=', 'draft'), ('origin', '=', op.name)]):
+            for pr in self.env['purchase.requisition'].search([('state', '=', 'draft'), ('reference', '=', op.name)]):
```

Comment:
This link may be broken now that the `reference` is editable after confirmation of the purchase agreement. Not sure if this actually currently consistently works / does anything though.

cc: @thomasdeleval @amoyaux

Reply:
Can be ignored for now, can be fixed in a bugfix later if it ends up being an issue (or a future code cleanup, if you want you can add a "TODO: check if this still does something" comment to the method for better traceability)